src/charm.py

```diff
@@ -241,7 +249,17 @@ def _pgbouncer_layer(self) -> Layer:
             A pebble configuration layer for as many charm services as there are available CPU
             cores
         """
-        pebble_services = {}
+        pebble_services = {
+            "logrotate": {
+                "command": "sh -c 'logrotate -v /etc/logrotate.conf; sleep 5'",
```

Comment:
Pebble errors out if the command exits too fast (1s) and I couldn't find a way to disable that behaviour. Forcing the rotation (-f) usually took more, but then rules would be ignored, so I've added sleep to stabilise pebble's checks.